proxy/src/main/java/com/wavefront/agent/ProxyCheckInScheduler.java

```diff
@@ -192,6 +195,9 @@ private AgentConfiguration checkin() {
             throw new RuntimeException("Aborting start-up");
           }
           break;
+        case 429:
+          //
+          break;
```

Comment:
same here, is this intended?

Reply:
yup, we are ignoring 429s on purpose. updated the comment